src/libraries/Common/src/System/Security/Cryptography/Asn1/AsnXml.targets

```diff
@@ -21,23 +20,25 @@
       XmlInputPaths="@(AsnXml)"
       OutputPaths="@(AsnXml -> '$(IntermediateOutputPath)asnxml\%(filename).cs')" />
 
-    <Exec Condition="'$(TargetOS)'=='windows'"
+    <Exec Condition="$([MSBuild]::IsOsPlatform('windows')) == 'true'"
```

Comment:
a nit: in other places, the prevalent pattern is: `Condition="'$(TargetsWindows)' == 'true'"` (and TargetsUnix below). Could that work here?

Reply:
To help me better understand correctly: I thought that `TargetOS` and `TargetsWindows` are about what target you are building for, not the actual OS that MSBuild is running on. Regardless of the OS we are targeting to build; we want to use the `fc` tool if we are building on WIndows.

Reply:
Your understanding matches mine.  Probably means that the "hey, I updated something" warnings just didn't fire when building Unix-y projects from Windows.

Reply:
Ah, correct. I didn't realized that we have `build on Windows, targeting Unix` support. :)